src/components/input-group/stories/framework-react.tsx

```diff
@@ -12,9 +13,10 @@ export const ReactDaikinInputGroup = createComponent({
 });
 
 export const metadata: Meta<DaikinInputGroupStoryArgs> = {
-  component: ({ ...props }: DaikinInputGroupStoryArgs) => (
+  component: ({ content, ...props }: DaikinInputGroupStoryArgs) => (
     <ReactDaikinInputGroup {...props}>
-      <ReactDaikinTextInput value="Value" />
+      {content === "TextInput" && <ReactDaikinTextInput value="Value" />}
+      {content === "Textarea" && <ReactDaikinTextarea />}
```

Comment:
Let's set value as we do in `TextInput` to test if `value` attribute works.

```suggestion
      {content === "Textarea" && <ReactDaikinTextarea value="Value" />}
```